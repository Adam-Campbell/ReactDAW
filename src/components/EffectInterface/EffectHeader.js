import React from 'react';

const EffectHeader = props => (
    <div className="effect__header-container">
        <h1 className="effect__title">{props.effectTitle}</h1>
    </div>
);

export default EffectHeader;