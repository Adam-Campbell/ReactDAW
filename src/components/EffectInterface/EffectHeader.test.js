import React from 'react';
import { shallow } from 'enzyme';
import EffectHeader from './EffectHeader';

test('renders correctly', () => {
    const component = shallow(
        <EffectHeader 
            effectTitle="Chorus"
        />
    );
    expect(component).toMatchSnapshot();
    const title = <h1 className="effect__title">Chorus</h1>
    expect(component.contains(title)).toEqual(true);
});