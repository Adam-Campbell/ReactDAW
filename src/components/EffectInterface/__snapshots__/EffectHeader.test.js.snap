// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="effect__header-container"
>
  <h1
    className="effect__title"
  >
    Chorus
  </h1>
</div>
`;
